backend/buildeR.Common/Services/EmailService.cs

```diff
@@ -11,22 +11,33 @@ namespace buildeR.Common.Services
     
     public class EmailService : IEmailService
     {
-        public Task SendEmailAsync(List<string> emails, string subject, string message)
+        public Task SendEmailAsync(List<string> emails, string subject, string title, string body)
         {
 
-            return Execute(Environment.GetEnvironmentVariable("SENDGRID_API_KEY"), subject, message, emails);
+            return Execute(Environment.GetEnvironmentVariable("SENDGRID_API_KEY"), emails, subject, title, body);
```

Comment:
Could you move getting all env variables into constructor and save them into class fields?